stable/nginx-ingress/templates/controller-daemonset.yaml

```diff
@@ -80,9 +80,15 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-          {{- range $key, $value := .Values.controller.extraEnvs }}
-            - name: {{ $key | upper | replace "." "_" }}
-              value: {{ $value }}
+          {{- if .Values.controller.extraEnvs }}
+            {{- if .Values.controller.inlineExtraEnvs }}
+{{ toYaml .Values.controller.extraEnvs | indent 12 }}
+            {{- else }}
+              {{- range $key, $value := .Values.controller.extraEnvs }}
+                - name: {{ $key | upper | replace "." "_" }}
```

Comment:
this looks like it's indented too too much.

Reply:
5f84084cc59de7b350a62ae754d5ffc53dbb0896